hook.sh

```diff
@@ -76,12 +85,12 @@ function deploy_challenge {
 
     # Even if the transaction is executed, the results may not be available in the DNS servers yet
     echo "Verifying results on live DNS servers:"
-    for nameserver in $(dig $dnsDomain NS +short); do 
-        echo -n "$nameserver " 
+    for nameserver in $(dig $dnsDomain NS +short); do
+        echo -n "$nameserver "
         nsresult=$(dig _acme-challenge.$DOMAIN TXT @$nameserver +short)
         # nsresult comes with the TXT RR in double quotes - remove those
         nsresult=${nsresult//$'"'/''}
-        until [[ "$nsresult" = "$TOKEN_VALUE" ]]; do
+        until [[ "$nsresult" == *"$TOKEN_VALUE"* ]]; do
```

Comment:
What is the difference between the old version and the new? Won't `*` expand in bash?